src/Apps/Checkout/ApplePay.tsx

```diff
@@ -0,0 +1,12 @@
+import { ExpressCheckoutQueryRenderer } from "Apps/Order/Components/ExpressCheckout"
+
+export const ApplePay = props => {
+  console.log("ApplePay.tsx", props)
+  // const { match } = useRouter()
+
+  // console.log("ApplePay.tsx", match.params.orderID)
+
+  // return null
+
+  return <ExpressCheckoutQueryRenderer orderID={props.match.params.orderID} />
```

Comment:
#minor - we should use the same criteria to decide whether to render this as we do in the legacy Shipping/index.tsx file (buy orders, feature flag, etc)